server/core/region.go

```diff
@@ -698,8 +701,11 @@ func GenerateRegionGuideFunc(enableLog bool) RegionGuideFunc {
 
 // RegionsInfo for export
 type RegionsInfo struct {
+	t            syncutil.RWMutex
 	tree         *regionTree
 	regions      map[uint64]*regionItem // regionID -> regionInfo
+	st           syncutil.RWMutex
+	subRegions   map[uint64]*regionItem // regionID -> regionInfo
```

Comment:
Use code like this to identify the scope:
```
	mu struct {
		syncutil.RWMutex
		streams      map[string]ServerStream
		clientCtx    context.Context
		clientCancel context.CancelFunc
	}
```

Reply:
The map also needs to be locked, so maybe wrap them later?